lib/logstash/plugin_mixins/jdbc/statement_handler.rb

```diff
@@ -101,20 +110,29 @@ class ExplicitPagingModeStatementHandler < PagedNormalStatementHandler
     def perform_query(db, sql_last_value)
       query = build_query(db, sql_last_value)
       offset = 0
+      page_size = @jdbc_page_size
```

Comment:
`@jdbc_page_size`is a not modifiable class field, it passed in constructor and never updated. I don't understand the need to copy into a local variable.

Reply:
the i-var won't be read in a loop repeatedly but most importantly it helped readability of the method since none of the other locals use the jdbc_ prefix and it looked a little "confusing" why one has the prefix but none of the others, now:
```ruby
        break unless rows_in_page == page_size
        offset += page_size
```
> it passed in constructor and never updated. I don't understand the need to copy into a local variable.

correct, again, mostly just refactored the `post_init` - it's a convention used through the whole file that plugin variables are copied to i-vars.